config.json

```diff
@@ -500,6 +500,17 @@
           "algorithms"
         ]
       },
+      {
+        "slug": "satellite",
+        "name": "Satellite",
+        "uuid": "22136812-8160-49f0-8a0d-f76a2086a34c",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 6,
+        "topics": [
+          "maybe"
+        ]
```

Comment:
@ErikSchierboom I am not familiar with this feature, so I cannot judge this.

Reply:
Topics are deprecated and are not used by the website. They are allowed to be in the `config.json` as they used to be used and they could be useful to help determine the `practices` and `prerequisites` fields.